03-setup-ingress-controller/nginx.md

```diff
@@ -744,16 +747,25 @@ Then, uncomment the `service` section as seen below:
 
 ```yaml
 service:
- type: LoadBalancer
- annotations:
-   # Enable proxy protocol
-   service.beta.kubernetes.io/do-loadbalancer-enable-proxy-protocol: "true"
+  type: LoadBalancer
+  annotations:
+    # Enable proxy protocol
+    service.beta.kubernetes.io/do-loadbalancer-enable-proxy-protocol: "true"
+    # Specify whether the DigitalOcean Load Balancer should pass encrypted data to backend droplets
+    service.beta.kubernetes.io/do-loadbalancer-tls-passthrough: "true"
 ```
 
 **Note:**
 
 You must **NOT** create a load balancer with `Proxy` support by using the `DigitalOcean` web console, as any setting done outside `DOKS` is automatically `overridden` by DOKS `reconciliation`.
 
+Then, uncomment the `config` section as seen below to allow Nginx to receive client connection information:
+
+```yaml
+config:
+  use-proxy-protocol: "true"
+```
+
 Finally, after saving the values file, you can apply changes using `Helm`:
 
```

Comment:
After applying the changes via Helm, I would add a small `curl` example showing the user that proxy protocol is enabled and it works. It means, the client should see own IP address in the response.